### PR TITLE
[BIG PR] Just the early stuff

### PR DESCRIPTION
**Pull request by thisisdano**
This is all the commits from December 8 in the big PR (https://github.com/18F/cg-style/pull/225/commits):

I don't think any of these changes had comments in the original PR.

screen shot 2016-12-21 at 10 19 46 am

### CODE REVIEW

**Comment by thisisdano** [comment]
ah crap — this needs a fix. one sec

File: src/css/components/panel.scss [hunk 1]
```diff
@@ -3,25 +3,16 @@
...
-  min-height: 6rem;
-  padding: .75rem 0 0;
+  min-height: 10rem;
+  padding: 0.75rem 0 0 0;
 
   & + .panel {
     margin-top: 1.5rem;
   }
-
-  table {
```
Comment (msecret) [inline src/css/components/panel.scss:17]: I think you might be missing some of the commits I recently put in here to modify the layouts on marketplace. How did you rebase this?

Comment (msecret) [reply src/css/components/panel.scss:17]: I think changes were here: https://github.com/18F/cg-style/commit/7282089bf8e101bf22bade720f9258de7d3f4f1b

Comment (thisisdano) [reply src/css/components/panel.scss:17]: ah crap — I screwed this up. I just pasted my file at old PR's commit point into file I had after updating master. So I blitzed yr changes. I'll go back and restore them.

Comment (thisisdano) [reply src/css/components/panel.scss:17]: OK, I think I fixed this

**Comment by thisisdano** [comment]
OK, I went back and added the new panel styles. I blitzed them by mistake when adding my changes to the branch.

**Comment by msecret** [comment]
This looks good once build and tests are fixed

**Comment by thisisdano** [comment]
OK, fixed I think...

**Review by line47 — APPROVED** [review]

**Pull request merged**